unittests/test_warnings.py

```diff
@@ -23,7 +23,9 @@ def test_deprecation_warning():
 
 def test_deprecation_warning_from_version():
```

Comment:
Is it possible to test a case where the a deprecation warning is thrown. Like test for a version previous to this one.

Reply:
The default `from_version` is `0.0.0`, so what you ask here is implicitly tested, isn't it?

Reply:
I mean a case where the deprecation warning is actually thrown.

Reply:
I think the `test_deprecation_warning` does already this.